README.md

```diff
@@ -105,6 +102,56 @@ export default function Dashboard({ data }: PageProps<Data>) {
 }
 ```
 
+## Usage Cookie Options
+
+session value is cookie. can set the option for cookie.
+
+```ts
+import { MiddlewareHandlerContext } from "$fresh/server.ts";
+import { createCookieSession, WithSession } from "fresh-session";
+
+export type State = {} & WithSession;
+
+const cookieSession = createCookieSession({
+  maxAge: 30, //Session keep is 30 seconds.
+  httpOnly: true,
+});
+
+export function session(req: Request, ctx: MiddlewareHandlerContext<State>) {
+  return cookieSession(req, ctx);
+}
+export const handler = [session];
```

Comment:
```suggestion
export const handler = [
  createCookieSession({
    maxAge: 30, //Session keep is 30 seconds.
    httpOnly: true,
  });
];
```

This way, the example is a little easier to implement for the end-user who doesn't need to reimplement a session middleware of top of our middleware.

I also made a comment on `createCookieSession` to use `cookieSession` directly instead.

If we decide to merge it, we would need to update it here as well